x-pack/platform/plugins/shared/alerting_v2/server/saved_objects/schemas/raw_esql_rule/v1.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+/**
+ * Attributes for the ES|QL rule saved object.
+ *
+ * Shape mirrors the external API contract (string schedule, esql, etc) and
+ * includes internal fields needed for task scheduling and credentials.
+ */
+export const rawEsqlRuleSchema = schema.object({
+  name: schema.string(),
+  tags: schema.arrayOf(schema.string(), { defaultValue: [] }),
+  schedule: schema.string(),
```

Comment:
```suggestion
  schedule: schema.object({ custom: schema.string() }), // schedule.custom = "5m"
```

Not sure about this suggestion, but I am wondering if we would need to support different schedule types in the future, like cron or rrule.